src/Pages/EventMain/EventPage.js

```diff
@@ -9,6 +9,8 @@ import { useHistory } from 'react-router-dom'
 export default function EventPage() {
   const [categoryFilter, setCategoryFilter] = useState('all')
   const [eventFilter, setEventFilter] = useState('date')
+  const [searchedEvent, setSearchedEvent] = useState('')
```

Comment:
I don't understand the two states "searchedEvent" and "searchedEvents". I thought "searchedEvents" should be an array, but it's a string too. Maybe you can think about the naming to make it more clear what the two states are for.

Reply:
at the moment it's one string and the other can be more strings, naming is difficult here, thats true